Authorization/Authorization/en.lproj/Localizable.strings

```diff
@@ -16,21 +16,34 @@
 "SIGN_IN.LOG_IN_BTN" = "Sign in";
 
 "ERROR.INVALID_EMAIL_ADDRESS" = "Invalid email address";
+"ERROR.INVALID_PASSWORD_LENGHT" = "Invalid password lenght";
+"ERROR.AUTH_PROVIDER" = "This %@ account is not linked with any %@ account. Please register.";
```

Comment:
How about renaming it to `ERROR.ACCOUNT_NOT_LINKED` OR `ERROR.ACCOUNT_NOT_REGISTERED`?